iot/iot-device-registry-base/src/main/java/io/enmasse/iot/registry/tenant/KubernetesTenantInformationService.java

```diff
@@ -73,7 +110,7 @@
      * @return The completed future, never returns {@code null}.
      */
     private static <T> Future<T> notFound(final int notFoundStatusCode, final String message) {
+        log.info(message);
```

Comment:
I don't think we should have "not found" in info, that might spam the logs a bit.